src/commands/analyze.js

```diff
@@ -123,13 +123,13 @@ export const handler = async (argv) => {
             })
         }
 
-        jobResult.passed = Object.values(jobResult.results).find((r) => !r.passed)
+        jobResult.passed = !Object.values(jobResult.results).find((r) => !r.passed)
```

Comment:
```suggestion
        jobResult.passed = Object.values(jobResult.results).find((r) => r.passed).length > 0
```

Reply:
Shouldn't it just be Object.values(jobResult.results).find((r) => r.passed)?

I think find just returns the first element, not a list.

Reply:
Actually that doesn't seem right still, we'd still just pass if we had any results that passed when we should only pass if they all pass.  We want to pass only if we don't find non-passes.  I think this is why the double negation was needed.

Reply:
Oh right. Sorry, I didn't notice that `find` is outside and the result is `undefined` or a single item. Confusing JS... So maybe `every()`?
```js
jobResult.passed = Object.values(jobResult.results).every((r) => r.passed)
```
or `some()` but it requires negation as well:
```js
jobResult.passed = !Object.values(jobResult.results).some((r) => !r.passed)
```